coderd/database/queries/notifications.sql

```diff
@@ -13,14 +13,15 @@ WHERE nt.id = @notification_template_id
   AND u.id = @user_id;
 
 -- name: EnqueueNotificationMessage :exec
-INSERT INTO notification_messages (id, notification_template_id, user_id, method, payload, targets, created_by)
+INSERT INTO notification_messages (id, notification_template_id, user_id, method, payload, targets, created_by, created_at)
```

Comment:
I added `created_at` to be passed in to make the dedupe hash testable by using Quartz to advance the clock.